README.rst

```diff
@@ -57,6 +57,39 @@ Questions?
 
 Join us in `#sopel <irc://irc.freenode.net/#sopel>`_ on Freenode.
 
+Credits
++++++++
+
+Contributors
+------------
+
+This project exists thanks to all the people who contribute! `Become a contributor`__.
+
+.. image:: https://opencollective.com/sopel/contributors.svg?width=890&button=false
+    :target: https://github.com/evhub/sopel/graphs/contributors
```

Comment:
Where did this repo come from? It doesn't exist.

Reply:
possibly a typo by me. I try not to start over with each RST because I usually write MD, so I copy the last one.

Reply:
```suggestion
    :target: https://github.com/sopel-irc/sopel/graphs/contributors
```